# deklariranje topica za senzore ovisno o tipu podatka

mqtt_bridge_node:
  ros__parameters:
    mqtt_host: "tcp://192.168.2.100:1883"
    mqtt_client_id: "ros2_mqtt_bridge"

    float64_topics:
      - "/depth"
      - "/depth_variance"
      - "/altitude"
      - "/altitude_covariance"
      - "/velocity_x"
      - "/velocity_y"

    float64_multiarray_topics:
      - "/velocity_covariance"
      - "/beam_ranges"
